tests/test_tree_distance.py

```diff
@@ -3,17 +3,86 @@
 import pytest
 
 from cogent3 import make_tree
-from cogent3.phylo.tree_distance import lin_rajan_moret
+from cogent3.phylo.tree_distance import (
+    lin_rajan_moret,
+    matching_cluster_distance,
+    rooted_robinson_foulds,
+    unrooted_robinson_foulds,
+)
+
+ROOTED_DISTANCE_MEASURES = (matching_cluster_distance, rooted_robinson_foulds)
+UNROOTED_DISTANCE_MEASURES = (lin_rajan_moret, unrooted_robinson_foulds)
+
+
+# unrooted rf distance
+def test_unrooted_rf_different_trees():
+    a = make_tree(treestring="(a, b, (c, (d, e)));")
+
+    b = make_tree(treestring="(a, b, (d, (c, e)));")
+    distance = unrooted_robinson_foulds(a, b)
+    assert distance == 2
+    distance = unrooted_robinson_foulds(b, a)
+    assert distance == 2
+
+    b = make_tree(treestring="((a, b), d, (c, e));")
+    distance = unrooted_robinson_foulds(a, b)
+    assert distance == 2
+    distance = unrooted_robinson_foulds(b, a)
+    assert distance == 2
+
+    b = make_tree(treestring="(((a, b), d), c, e);")
+    distance = unrooted_robinson_foulds(a, b)
+    assert distance == 2
+    distance = unrooted_robinson_foulds(b, a)
+    assert distance == 2
+
+    a = make_tree(treestring="(a, (b, (c, d)), (e, (f, (g, h))));")
+    b = make_tree(treestring="(a, (c, (b, d)), (h, (f, (g, e))));")
+    distance = unrooted_robinson_foulds(a, b)
+    assert distance == 6
+    distance = unrooted_robinson_foulds(b, a)
+    assert distance == 6
+
+
+def test_unrooted_rf_same_tree():
+    a = make_tree(treestring="(a, b, (c, d));")
```

Comment:
can you add a comment that by same tree you mean the same topology, but a different "orientation"